.metadocrc

```diff
@@ -0,0 +1,15 @@
+{
+	"files": [
```

Comment:
I think you should replace tabs with spaces in this file since we don't use tabs for indentation.